src/cargo/ops/cargo_install.rs

```diff
@@ -209,8 +209,10 @@ impl<'cfg, 'a> InstallablePackage<'cfg, 'a> {
             bail!(
                 "there is nothing to install in `{}`, because it has no binaries\n\
                  `cargo install` is only for installing programs, and can't be used with libraries.\n\
-                 To use a library crate, add it as a dependency in a Cargo project instead.",
-                pkg
+                 To use a library crate, add it as a dependency in a Cargo project instead.\n\
+                 Use `cargo add {}` if you want to simply add it as a dependency of the current Cargo project.",
```

Comment:
Should we merge these two lines:
```
To use a library crate, add it as a dependency to a Cargo project with `cargo add {}`.
```

The more verbose we get, the more likely someone will will have stopped reading earlier.